src/main/java/com/zendesk/maxwell/RowMap.java

```diff
@@ -105,7 +105,7 @@ private String pkToJsonHash() throws IOException {
 				if ( data.containsKey(pk) )
 					pkValue = data.get(pk);
 
-				g.writeObjectField("pk." + pk, pkValue);
+				g.writeObjectField("pk." + pk.toLowerCase(), pkValue);
```

Comment:
so the change is:

- Columns now are CaSed
- Primary keys are still output as lower_cased

so in order to get the value for a PK row, the PK is stored inside the `Table` object as "CaSed" (by `normalizedPKColumns`), looked up in `RowMap#data` as "CaSed", but then output as "cased".